impressionist/readme.txt

```diff
@@ -0,0 +1,55 @@
+=== Impressionist ===
+Contributors: Automattic
+Requires at least: 6.0
+Tested up to: 6.6
+Requires PHP: 5.7
+License: GPLv2 or later
+License URI: http://www.gnu.org/licenses/gpl-2.0.html
+
+== Description ==
+
+Impressionist is a block theme about the impressionist art movement.
+
+== Changelog ==
+
+= 1.0.0 =
+* Initial release
+
+== Copyright ==
+
+Impressionist is based on Impressionist (https://github.com/Automattic/themes/tree/trunk/impressionist), (C) Automattic, [GPLv2 or later](http://www.gnu.org/licenses/gpl-2.0.html)
+
+Impressionist WordPress Theme, (C) 2022 Automattic
+Impressionist is distributed under the terms of the GNU GPL.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+This theme bundles the following third-party resources:
+
+Beth Ellen Font
+Copyright 2019 The Beth Ellen Project Authors (https://github.com/googlefonts/BethEllen) 
+This Font Software is licensed under the SIL Open Font License, Version 1.1. This license is available with a FAQ at: http://scripts.sil.org/OFL 
+License URL: http://scripts.sil.org/OFL 
+-- End of Beth Ellen Font credits --
+
+Cardo Font
+Copyright (c) 2002-2011, David J. Perry (hospes02@scholarsfonts.net) 
+This Font Software is licensed under the SIL Open Font License, Version 1.1. This license is available with a FAQ at: http://scripts.sil.org/OFL  
+Source: http://scholarsfonts.net
+-- End of Cardo Font credits --
+
+impressionist-home-bg.jpg
+CC0, Public Domain
+https://www.rawpixel.com/image/2677419/free-illustration-image-winter-monet-snow
```

Comment:
We probably want to change the format of this, since the .org reviewers always ask for it to be something like:

```
Credit Jakub Kapusnak
License: Creative Commons Zero (CC0) license
License URL: https://www.foodiesfeed.com/license/
Source: https://www.foodiesfeed.com/free-food-photo/sea-crab-at-fish-market/
```

Reply:
And also say where the images are used, including the theme screenshot

Reply:
Good call. I've edited it before the submission.